com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/GameProjectBuilder.java

```diff
@@ -118,8 +122,10 @@ private RandomAccessFile createRandomAccessFile(File handle) throws IOException
             }
         }
 
-        project.buildResource(input, CopyCustomResourcesBuilder.class);
-
+        // ignore custom resources if processing a project file other than game.project
```

Comment:
This was the cause of the reported problem. If two *.project files both specified the same custom_resources folder the files would get added twice and trigger the "Conflicting output resource" error